### PR TITLE
rework the aws cli docs

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: content/en/user-guide/integrations/aws-cli/index.md [hunk 1]
```diff
@@ -37,71 +39,75 @@ $ export AWS_DEFAULT_REGION="us-east-1"
...
-Verify the current configuration:
+### Configuring a custom profile
+
+You can configure a custom profile to use with LocalStack. Add the following profile to your AWS configuration file (by default, this file is at `~/.aws/config`):
+
+```bash
+[profile localstack]
+region=us-east-1
+output=json
+endpoint_url = http://localhost:4566
```
Comment (whummer) [inline content/en/user-guide/integrations/aws-cli/index.md:54]: 🥳

File: content/en/user-guide/integrations/aws-cli/index.md [hunk 2]
```diff
@@ -37,71 +39,75 @@ $ export AWS_DEFAULT_REGION="us-east-1"
...
+aws_secret_access_key=test
+```
+
+You can now use the `localstack` profile with the `aws` CLI:
 
 {{< command >}}
-aws configure list
+$ aws s3 mb s3://test --profile localstack
+$ aws s3 ls --profile localstack
 {{< / command >}}
```
Comment (whummer) [inline content/en/user-guide/integrations/aws-cli/index.md:71]: Maybe add a note here, similar to:

> Alternatively, you can also set the `AWS_PROFILE=localstack` environment variable, in which case the `--profile localstack` parameter can be omitted in the commands above.

File: content/en/user-guide/integrations/aws-cli/index.md [hunk 3]
```diff
@@ -37,71 +39,75 @@ $ export AWS_DEFAULT_REGION="us-east-1"
...
-Verify the current configuration:
-
-{{< command >}}
-awslocal configure list
-{{< / command >}}
+### Configuration
 
+| Variable         | Description                                                                                                   |
+| ---------------- | ------------------------------------------------------------------------------------------------------------- |
+| `LOCALSTACK_HOST` | Specifies the hostname for the LocalStack instance, particularly useful when LocalStack is bound to a different network interface, such as a docker-machine. |
```
Comment (whummer) [inline content/en/user-guide/integrations/aws-cli/index.md:108]: I think those are a bit outdated - the new canonical way to configure the target endpoint is via `AWS_ENDPOINT_URL`, and the `LOCALSTACK_HOST`/`USE_SSL` configs are deprecated. More details here: https://github.com/localstack/awscli-local#configurations

**Comment by HarshCasper** [comment]
After a discussion with @MarcelStranak, we will discuss refactoring of the AWS CLI v2 docs next week. I will merge this PR to give way to some new docs where the AWS profile needs to be mentioned.

**Pull request merged**